.github/workflows/weekly_updates.yml

```diff
@@ -16,13 +16,30 @@ jobs:
     name: Create post on Make site
     runs-on: ubuntu-latest
     steps:
+      - uses: pnpm/action-setup@v2.2.4
+        with:
+          # Ensure that it matchers `packageManager` field.
+          version: 7.17.1
```

Comment:
Can we use the same line as the frontend Playwright setup linked below to get the correct version of pnpm?
https://github.com/WordPress/openverse-frontend/blob/3baca4c90b1f3912bbd8058fb316e416d522a94a/Dockerfile.playwright#L6

Reply:
Too late, I opened this tab yesterday and didn't refresh before posting :)

Reply:
Oh no! This was very valuable feedback. Could you make a new issue, or PR, to implement this?